src/components/Login/Login.js

```diff
@@ -18,12 +19,17 @@ const Login = () => {
     const navigate = useNavigate();
     const [email, setEmail] = useState('');
     const [password, setPassword] = useState('');
-
-    //Submit Handler
+    const { userData, userToken, setUserData, setUserToken } = useAuthContext();
     const submitHandler = (event) => {
         event.preventDefault();
-        auth.login(email, password);
+
+        Auth(email, password);
+        setUserData(localStorage.getItem('userData'));
+        setUserToken(localStorage.getItem('userToken'));
```

Comment:
This part of the logic should be inside the AuthContext - AuthContext should provide a function `login` that calls the Auth service, and if login is successful, sets its userDatra and userToken state. Reading userData and token from localStorage should also happen inside of AuthContext, but inside useEffect, so we only execute that once, when the app is first initialized

Also, AuthContext should redirect to the user's profile page if the login is successful